pkg/kubectl/cmd/cmd.go

```diff
@@ -110,23 +110,26 @@ __custom_func() {
 }
 `
 	valid_resources = `Valid resource types include:
-   * pods (aka 'po')
-   * replicationcontrollers (aka 'rc')
+   * componentstatuses (aka 'cs')
    * daemonsets (aka 'ds')
-   * services (aka 'svc')
+   * deployments
```

Comment:
Since we don't include all resources here, you should probably remove the experimentals. A user may be aware of their existence (docs as you mentioned) but chose not to run them and `kubectl get` would still show him resources she/he isn't interested in.